transactional_licence_form/forms.py

```diff
@@ -0,0 +1,317 @@
+        label_class="as_para",
+    )
+    licence_holder_lastname = FCLCharField(
+        label="The full name of the person in your organisation who will be responsible for licence compliance",
+        max_length=50,
+        show_hidden_initial=True,
+    )
+    licence_holder_email = FCLCharField(
+        label="The email address of the person in your organisation who will be responsible for licence compliance",
+        max_length=50,
+        show_hidden_initial=True,
+    )
+
+
+class OrganizationForm(forms.Form):
+
+    TNA_CONTACTTYPE_CHOICES = [
+        "Private limited company",
+        "Public limited company",
+        "Partnership",
+        "Sole trader",
+        "Registered Charity",
+        "Community interest company",
+        "Independent research organisation",
+        "Public body",
+        "Independent body",
+        "Other (please specify)",
+    ]
+
+    agent_companyname = FCLCharField(
+        label="What is the full legal name of your organisation?", max_length=50
+    )
+    agent_companyname_other = FCLCharField(
+        label="Please enter any other names your organisation is known by",
+        max_length=50,
+        help_text="If your organisation is not known by any other names, please type <strong>none</strong>.",
+    )
+    agent_country = FCLCharField(
+        label="Where is your organisation registered?"
+    )  # TODO Tim make this an autocomplete
+
+    tna_contacttype = FCLMultipleChoiceField(
+        label="What type of organisation is it?",
+        help_text="Select all that apply.",
+        widget=forms.CheckboxSelectMultiple,
+        choices=list_to_choices(TNA_CONTACTTYPE_CHOICES),
+    )
+    org_other = FCLCharField(max_length=50)
+    agent_companyid = FCLCharField(
+        max_length=100,
+        label="Please provide your organisation identifier (e.g. company number or charity registration number)",
+        help_text="If your organisation does not have an identifier, please type <strong>none</strong>.",
+    )
+    partners = FCLCharField(
+        max_length=100,
+        label="Please provide the name of any partners or organisations you are working with",
+        help_text="If you are not working with any partners or organisations, please type <strong>none</strong>.",
+    )
+
+
+class ProjectPurposeForm(forms.Form):
+
+    PROJECT_PURPOSE_CHOICES = [
+        "Publish legal information",
+        "Produce summaries and interpretation of the records",
+        "Research and develop new technologies",
+        "Research activity and trends across records",
+        "Deliver a consumer service",
+        "Other (please specify)",
+    ]
+
+    USER_CHOICES = [
+        "Public access (e.g. anyone can freely access)",
+        "Restricted access (e.g. only subscribers or research peers)",
+        "Internal access (e.g. only colleagues from within your organisation)",
+        "Private for personal use only",
+    ]
+
+    BENEFIT_CHOICES = [
+        "General public",
```

Comment:
There's a part of me considering whether these would be happier in a separate JSON file, but it's fine as is.